tests/ParentClassResolver/Fixture/SomeClassWithRootNamespaceParent.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\SwissKnife\Tests\ParentClassResolver\Fixture;
+
+final class SomeClassWithRootNamespaceParent extends \SomeUnknownRootNamespaceClass
```

Comment:
We only work with existing classes, that min required level to use this tool.

Reply:
for the test-case it doesn't matter whether the class exists or not. it only matter whether the class is in the root namespace.

I did not add a class withthin root namespace, as this would possible collide with other test-cases. therefore I just used a "non-existant" root class, which works the same way without the risk of colliding